server/src/api/modules/groups/services/EditGroupService.ts

```diff
@@ -448,3 +479,20 @@ function calculateRoleChangeMaps(
 }
 
 export { editGroup };
+
+async function updateGroupRoleOrder(
+  groupId: number,
+  roleOrders: Array<{ role: GroupRole; index: number }>,
+  transaction: PrismaTypes.TransactionClient
+) {
+  // TODO: unsure if there's a better way to delete/replace records using prisma
```

Comment:
This approach seems fine to me, it's running inside a transaction anyways

Reply:
removed todo